engines/sci/engine/scriptdebug.cpp

```diff
@@ -256,7 +256,14 @@ reg_t disassemble(EngineState *s, reg_t pos, const Object *obj, bool printBWTag,
 					separator = ", ";
 				}
 
-				if (opcode == op_pushi && param_value < kernel->getSelectorNamesSize()) {
+				// Give additional selector name context for all number push scenarios.
+				if (opcode == op_push0) {
```

Comment:
You could simplify this block as follows:

    if (opcode >= op_push0 && opcode <= op_push2) {
        debugN("%s%s", separator, kernel->getSelectorName(opcode - op_push0).c_str());
    }

Reply:
@bluegr - very nice! I’ll update it to this. Thank you for the review.